_data/projects/react-test.yml

```diff
@@ -0,0 +1,16 @@
+name: React Test
```

Comment:
@franciscop Can we change the name to something else so that users will not think that this is Official https://reactjs.org/ product.

Reply:
Oh I didn't think about it getting confused because it's very common to do `react-X` in the community for 3rd party packages, but of course if it's just `react-test` it could easily be confused! I'll update the project site and everything as well, thanks for the input!